.github/workflows/gh_actions_push_next.yaml

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2012-2020 Red Hat, Inc.
```

Comment:
fix copyright to 2021; please also rename this file to be consistent w/ other repos that publish a next image.